test/test.js

```diff
@@ -112,6 +115,20 @@ test('json', async (t) => {
   delete stats.chunks;
   delete stats.modules;
 
+  // Remove properties that aren't common in webpack 4 and 5
+  delete stats.assets;
```

Comment:
we need to be checking `assets`, `assetsByChunkName`, `errors`, `entryPoints`, `warningsCount`, `publicPath` because they assert the bundle is correct.

Reply:
Ok, let's fork the tests for wp 4/5 here then for handling this case. I'll add test files just for this.

Reply:
We shouldn't need to. export the `isWebpack5` function from the index and use it here. You can manually compare using `is/deepEqual` or check for values on the object rather than snapshot.

Reply:
I found a way to do partial snapshots for 4/5. That feels a notch simpler.